external/skiboot/libstb/secvar/crypto/crypto-mbedtls.c

```diff
@@ -14,7 +14,6 @@
 #include "external/extraMbedtls/include/pkcs7.h"
 #include "external/extraMbedtls/include/generate-pkcs7.h"
 #include <mbedtls/platform.h>
-#include "generic.h"
```

Comment:
Yeah this is a good start. I wonder if we should wait until you have all the parts to get this to compile purely as part of skiboot and then go from there?